src/app/app-store/actions/counter-actions.ts

```diff
@@ -1,29 +1,39 @@
 import { Action } from '@ngrx/store';
 //create entity: import { Book } from './book-model';
 
-export const INCREMENT:string = "INCREMENT";
-export const INCREMENT_BY:string = "INCREMENT_BY";
-export const EVENT_FROM_EFFECT:string = "EVENT_FROM_EFFECT";
+export const INCREMENT = 'INCREMENT';

```

Comment:
Removing the `:string` amazingly resolves the `payload` compilation error we were encountering.